README.md

```diff
@@ -125,7 +125,7 @@ Manual Pages
 The manual pages for the master branch and all current stable releases are
 available online.
 
-- [OpenSSL master](https://www.openssl.org/docs/manmaster)
+- [OpenSSL master](https://www.openssl.org/docs/man3.0)
```

Comment:
Oops, this is wrong - should you add the 3.0 link instead?

Reply:
Ah, good catch!  My fault, it's the result of a recursive sed replace, and I didn't notice it when looking at the diff.

Reply:
done.

Reply:
Btw, this should be done in master as well, shouldn't it? Would you open a separate PR?

Reply:
Done, see #17855.